src/css/mapbox-gl.css

```diff
@@ -1,8 +1,8 @@
 .mapboxgl-map {
-    font: 12px/20px 'Helvetica Neue', Arial, Helvetica, sans-serif;
+    font: 12px/20px "Helvetica Neue", Arial, Helvetica, sans-serif;
     overflow: hidden;
     position: relative;
-    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    -webkit-tap-highlight-color: rgb(0 0 0 / 0%);
```

Comment:
Where does that weird % notation come from, newer stylelint? I've never seen that before.

Reply:
Yep, these are from updating stylelint. It appears that the % part isn't strictly necessary to satisfy the [rule](https://stylelint.io/user-guide/rules/list/color-function-notation/#:~:text=Modern%20color%2Dfunctions%20use%20a,used%20to%20separate%20repetitions%20only.), but the lint message called out using % so that's how I updated it.